Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.5
+
+MAINTAINER Zappi DevOps <devops@zappistore.com>
+
+ARG APP_DEPS="python py-setuptools"
```

Comment:
Before anyone asks, `py-setuptools` is necessary to avoid the below runtime error:

```
/ # curator --help
Traceback (most recent call last):
  File "/usr/bin/curator", line 6, in <module>
    from pkg_resources import load_entry_point
ImportError: No module named pkg_resources
```